actions/info.action.ts

```diff
@@ -50,16 +50,16 @@ export class InfoAction extends AbstractAction {
   }
 
   private displayBanner() {
-    console.info(chalk.red(BANNER));
+    console.info(red(BANNER));
   }
 
   private async displaySystemInformation(): Promise<void> {
-    console.info(chalk.green('[System Information]'));
+    console.info(green`[System Information]`);
     console.info(
       'OS Version     :',
-      chalk.blue(osName(platform(), release()), release()),
+      blue(osName(platform(), release()) + release()),
```

Comment:
Ansis method accepts only one argument as a string, therefore we need to concatenate two stings into one